src/datepicker/datepicker-day-view.spec.ts

```diff
@@ -0,0 +1,77 @@
+import {TestBed} from '@angular/core/testing';
+
+import {Component} from '@angular/core';
+import {NgbDatepickerModule} from './datepicker.module';
+import {NgbDatepickerDayView} from './datepicker-day-view';
+
+function getElement(element: HTMLElement): HTMLElement {
```

Comment:
Could you rename it to sth more descriptive? What kind of an element we are trying to get?